### PR TITLE
T 25 Poster. Calendar view

### PR DESCRIPTION
**Pull request by mAndrushchenko**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -44,6 +44,7 @@ import { SliderComponent } from './components/home-page/slider/slider.component'
 import { NewsListItemComponent } from './components/partials/news-list-item/news-list-item.component';
 
 import { environment } from '../environments/environment';
+import { CalPopupComponent } from './components/schedule/calendar/cal-popup/cal-popup.component';
```
Comment (yana-tsylinska) [inline src/app/app.module.ts:47]: Don't use such strange short names, right full, please

File: src/app/components/schedule/calendar/cal-popup/cal-popup.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  styleUrls: ['./cal-popup.component.scss']
+})
+export class CalPopupComponent {
+  @Input() date_time: string;
+  @Input() title: string;
+  @Input() time: string;
+  @Input() venue: string;
+  @Input() url: string;
+  @Input() img_title: string;
+  @Input() duration: string;
```
Comment (yana-tsylinska) [inline src/app/components/schedule/calendar/cal-popup/cal-popup.component.ts:15]: why??? it should be done better, max 1 Input

**Review by yana-tsylinska — CHANGES_REQUESTED** [review]

File: src/app/app.module.ts (hunk 1 above)
Comment (mAndrushchenko) [reply src/app/app.module.ts:47]: Ohh, I'll fix this in the next commit)

File: src/app/components/schedule/calendar/cal-popup/cal-popup.component.ts (hunk 2 above)
Comment (mAndrushchenko) [reply src/app/components/schedule/calendar/cal-popup/cal-popup.component.ts:15]: First of all, I used one Input with object, which had all of this properties. But when I tried to use it in component I have the [Object object] as the result. So, I decided to pass values like a 'string' and it solved my problem. But I understand that my solution is't  the best practice... What I did wrong? How I can solve this problem in another way?

File: src/assets/styles/_variables.scss [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-brake the code)*/
-
 $main-font: "Source Sans Pro", sans-serif;
 $white-color: #fff;
 $black-color: #000;
 $main-color-lighter: #fa231c;
+
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;700&display=swap");
+
+$Montserrat: montserrat;
```
Comment (yana-tsylinska) [inline src/assets/styles/_variables.scss:8]: check `$body-font-family`

**Review by yana-tsylinska — CHANGES_REQUESTED** [review]

**Comment by yana-tsylinska** [comment]
when the popup is opened on the right side, we're getting scroll in the bottom - https://prnt.sc/12nye2b
+ due to the fact that popup always appears above the performance title, we got this kind of bad behaviour - https://prnt.sc/12nyg3m

For now, you can merge it, but research tooltip/popup packages with flexible(auto-calculated) positioning - 'appears on that side, where there is enough space for it's content'

**Pull request closed without merging**